examples/holoclean_repair_example.py

```diff
@@ -1,7 +1,8 @@
 import holoclean
-from detect import NullDetector, ViolationDetector
+from detect import NullDetector, ViolationDetector, FixedDetector
```

Comment:
Please remove it from the example, it is a very special case.

Reply:
It was just for testing. Removed.